gateway-service-impl/src/main/java/org/hypertrace/gateway/service/explore/RequestHandler.java

```diff
@@ -69,6 +96,8 @@ QueryRequest buildQueryRequest(
       requestContext.setHasGroupBy(true);
     }
 
+    Optional<List<String>> maybeEntityIds = buildEntityIdFilterIfNecessary(requestContext, request);
+    LOG.info("EDS got the entity id as {}", maybeEntityIds);
```

Comment:
We should change this to DEBUG

Reply:
These all are there for testing. I will remove this.